rakelib/plugin.rake

```diff
@@ -86,9 +87,15 @@ namespace "plugin" do
 
     Rake::Task["plugin:clean-local-core-gem"].invoke(name, path)
 
-    puts("[plugin:build-local-core-gem] Building #{File.join(path, name)}.gemspec")
+    gemspec = "#{File.join(path, name)}.gemspec"
 
-    system("cd #{path}; gem build #{name}.gemspec")
+    puts("[plugin:build-local-core-gem] Building #{gemspec}")
+
+    spec = Gem::Specification.load gemspec
```

Comment:
any reason for omitting parenthesis on method call? this is not really a dsl context here, same comment below